src/StaticFAQ/FAQArticleDetail.js

```diff
@@ -37,14 +52,23 @@ class FAQArticleDetail extends React.Component<Props> {
       return <div>{params.error.message}</div>;
     }
     if (params.props) {
-      return <FAQArticleDetailContent article={params.props.FAQArticle} />;
+      return (
+        <FAQArticleDetailContent
+          category={params.props.FAQCategory}
+          article={params.props.FAQArticle}
+        />
+      );
     }
 
     return <Loader />;
   };
 
   render() {
     const articleId = idx(this.props.match, _ => _.params.articleId);
+    const categoryId = this.props.history.entries
```

Comment:
Although it should always work, I'm not entirely satisfied with this part. Do you have a suggestion that might improve it? :-)

Reply:
I would definitely suggest changing the url so it includes `categoryId` , e.g. `/categories/{categoryId}/articles/{articleid}`